src/partitioner/stream/Partition.cpp

```diff
@@ -61,6 +62,10 @@ double Partition::getEdgesCount() {
     double total = 0;
     std::set<std::string> uniqueEdges;
     for (auto edge : this->edgeList) {
+        std::string vertex1 = edge.first;
+        for (const auto& vertex2 : edge.second) {
+            std::cout << "Edge: (" << vertex1 << ", " << vertex2 << ")" << std::endl;
```

Comment:
Lets remove the for loop along with this `std::cout`. It does not do any productive work.

Reply:
Done.

Reply:
Ok